### ISSUE TITLE
doctest is failed by "ValueError: wrapper loop when unwrapping _"

### ISSUE THREAD

**Issue by katoken-0215** [fnpy/fn.py#32]
Hi, I tried to introduce current code which using fn.py, but I faced title error.

How to reproduce:

```
# (Install fn.py)
echo 'from fn import _' > test.py
python -m doctest test.py
```

My environment is Python 3.6.4 on macOS 10.13.3 installed by pyenv.

**Comment by katoken-0215** [comment]
I fixed #33 .
Please review !

**Comment by jacobbridges** [comment]
This pull request has been merged in.
